pypiper/utils.py

```diff
@@ -543,6 +543,11 @@ def _determine_args(argument_groups, arguments, use_all_args=False):
     :rtype: Set[str]
     """
 
+    if sys.version_info < (3, 3):
+        from collections import Iterable
+    else:
+        from collections.abc import Iterable
+
```

Comment:
why did they change this?

Reply:
I'm not totally sure why it's a subpackage now. It seems like this could slowly shift the way in which use of collections is considered, maybe encouraging thinking in ways like "what behaviors do I want/need to support?" more often than "what structure do I need to back this?" That's total speculation, though. Maybe also just a general trend toward more modularity in the core libraries.